### ISSUE TITLE
[Feature] weapon bob amount is 1/2 of normal when walking

### ISSUE THREAD

**Issue by lafoxxx92** [coelckers/prboom-plus#370]

**Comment by fabiangreffrath** [comment]
Why should we?

**Comment by srludicolo** [comment]
I think he's referring to an animation change in Smooth Doom, where the weapon bob plays at half the normal speed, but I dunno for sure because I do not use GZDoom

**Comment by lafoxxx92** [comment]
zdoom: movebob 0.125 and see how differently the gun moves when you walk/run

**Comment by fabiangreffrath** [comment]
I don't think that PrBoom+ is the right port to request this kind of feature. To be honest, this appears to me mostly like a random and subjective change of game mechanics.
